src/command_modules/azure-cli-container/azure/cli/command_modules/container/tests/test_container_commands.py

```diff
@@ -0,0 +1,102 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, JMESPathCheckExists, ResourceGroupPreparer)
+
+
+class AzureContainerInstanceScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_container_create(self, resource_group, resource_group_location):
+        '''Test create container with image, os type, ip address type, port, cpu, memory,
```

Comment:
Please avoid using doc string in the test method, this will interrupt the automation process. Please use `#` based comments.